File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Compra.kt
package ec.edu.monster.modelo

import java.io.Serializable
import java.util.Date

data class Compra(
    val idCompra: Int,
    val idVuelo: Int,
    val idCliente: Int,
    val vuelo: Vuelo? = null,
    val cliente: Cliente? = null,
    val fechaCompra: Date
) : Serializable

File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLICON/VIAJECITOS_REST_CLICON_GR10/VIAJECITOS_REST_CLICON_GR10/Modelos/Compra.cs
using System;

namespace VIAJECITOS_REST_CLICON_GR10.Modelos
{
    public class Compra
    {
        public int IdCompra { get; set; }
        public int IdVuelo { get; set; }
        public int IdCliente { get; set; }
        public DateTime FechaCompra { get; set; }
        public Vuelo Vuelo { get; set; }
        public Cliente Cliente { get; set; }
    }
}